.semaphore/semaphore.yml

```diff
@@ -0,0 +1,34 @@
+version: v1.0
+name: Apple/Mojave Based Pipeline
+agent:
+  machine:
+    type: a1-standard-4
+    os_image: macos-mojave-xcode11
```

Comment:
blorp!

> This is a reminder notification that the Xcode 11 macOS Mojave image will be deprecated tomorrow, on June the 5th.
> 
> In order to keep your projects up an running, please update your project configuration to use the latest macOS Catalina image, see below an example yaml::
>
> name: First Apple Pipeline
> agent:
>   machine:
>     type: a1-standard-4
>     os_image: macos-xcode11
> 
> For more info on Semaphore updates, please check the Semaphore Changelog.
> 
> Happy Building!
> The Semaphore Team
>

Reply:
Hopefully resolved with 7751266. Sorrys, i did not respond with rocket-ship speed. But I'm only like 1 or 2 days late 😝